manager/manager.go

```diff
@@ -712,6 +714,17 @@ func (m *Manager) watchForKEKChanges(ctx context.Context) error {
 	return nil
 }
 
+func (m *Manager) updateRootCA(ctx context.Context, cluster *api.Cluster) {
+	ctx = log.WithFields(ctx, logrus.Fields{
+		"cluster.id": cluster.ID,
+		"method":     "(*Manager).updateRootCA",
+	})
+	if err := ca.UpdateSecurityConfigFromCluster(ctx, m.config.SecurityConfig, cluster); err != nil {
+		log.G(ctx).WithError(err).Error("updating Root CA failed")
+	}
+	m.caserver.UpdateJoinTokens(ctx, cluster)
```

Comment:
Would a rename to something more appropriate be ok, or would you suggest having 2 watches on the cluster, one to update the root CA and one to update the join tokens?  My preference is for the first, to make sure that both are updated in sync.